backend/src/errors.rs

```diff
@@ -0,0 +1,13 @@
+error_chain! {
+    links {
+    }
+
+    foreign_links {
+        ::std::io::Error, Io, "io error";
+        ::json::Error, Json, "json error";
+        ::chrono::ParseError, Chrono, "chrono parsing error";
```

Comment:
I'm not sure that something so powerful as `error-chain` is needed here, or that I'm using it right (the strings especially seem "extra" to me since they're not really adding information.